relayer/relayPackets.go

```diff
@@ -166,7 +166,7 @@ func (rp *relayMsgRecvPacket) Msg(src, dst *Chain) (sdk.Msg, error) {
 	if rp.dstComRes == nil {
 		return nil, fmt.Errorf("receive packet [%s]seq{%d} has no associated proofs", src.ChainID, rp.seq)
 	}
-	version := clienttypes.ParseChainID(src.PathEnd.ChainID)
+	version := clienttypes.ParseChainID(src.ChainID)
```

Comment:
The relayer always confuses me, but I don't think we verify the path end chain id often and it is unclear to me if it is the chain id of the chain the client is on or the chain id of the client itself. We should just rely on the chain id associated with the light client here

Reply:
this becomes problematic if the chain-ids are swapped in the config for path end